isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -181,13 +181,19 @@ namespace Isis {
     p_hasAngularVelocity = rotToCopy.p_hasAngularVelocity;
     m_frameType = rotToCopy.m_frameType;
 
+    m_orientation = new ale::Orientations;
+    *m_orientation = *rotToCopy.m_orientation;
```

Comment:
This will seg fault if rotToCopy.m_orientation is null

Reply:
Thanks for catching this!